install.sh

```diff
@@ -29,27 +24,20 @@ wget https://github.com/ReFirmLabs/binwalk/archive/refs/tags/v2.3.3.tar.gz && \
   sed -i 's/^install_unstuff//g' deps.sh && \
   echo y | ./deps.sh && \
   sudo python3 setup.py install
-sudo apt-get install -y mtd-utils gzip bzip2 tar arj lhasa p7zip p7zip-full cabextract fusecram cramfsswap squashfs-tools sleuthkit default-jdk cpio lzop lzma srecord zlib1g-dev liblzma-dev liblzo2-dev unzip
+sudo apt-get install -y 
 
 cd - # back to root of project
 
 sudo cp core/unstuff /usr/local/bin/
 
-python3 -m pip install python-lzo cstruct ubi_reader
-sudo apt-get install -y python3-magic openjdk-8-jdk unrar
 
 # for analyzer, initializer
-sudo apt-get install -y python3-bs4
-python3 -m pip install selenium
 wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
 sudo dpkg -i google-chrome-stable_current_amd64.deb; sudo apt-get -fy install
 rm google-chrome-stable_current_amd64.deb
```

Comment:
Probably a good idea to add `git clone https://www.github.com/threat9/routersploit ./analysis/routersploit`